src/naksu/naksu.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const version = "1.5.0"
-const lowDiskLimit = 5000000 // 5 Gb
+const lowDiskLimit = 50000000 // 50 Gb
```

Comment:
I'd use `const lowDiskLimit = 50 * 1000 * 1000` or `const lowDiskLimit = 50 * 1024 * 1024` here for readability